### PR TITLE
MINOR: Minor improvements in consumer close timeout handling

### PR DESCRIPTION
**Pull request by hachikuji**

### CODE REVIEW

**Comment by hachikuji** [comment]
cc @ijuma @rajinisivaram

**Comment by rajinisivaram** [comment]
@hachikuji Thank you for the cleanup. Changes look good to me.

File: clients/src/main/java/org/apache/kafka/clients/consumer/internals/AbstractCoordinator.java [hunk 1]
```diff
@@ -647,21 +658,10 @@ protected synchronized void close(long timeoutMs) {
         // interrupted using wakeup) and the leave group request which have been queued, but not
         // yet sent to the broker. Wait up to close timeout for these pending requests to be processed.
         // If coordinator is not known, requests are aborted.
-        long now = time.milliseconds();
-        long endTimeMs = now + timeoutMs;
         Node coordinator = coordinator();
-        while (coordinator != null && client.pendingRequestCount(coordinator) > 0) {
-            if (Thread.currentThread().isInterrupted())
-                throw new InterruptException("Consumer close was interrupted");
```
Comment (ijuma) [inline clients/src/main/java/org/apache/kafka/clients/consumer/internals/AbstractCoordinator.java:655]: Seems like you removed the isInterrupted check, what's the reason?

**Review by ijuma — COMMENTED** [review]
Thanks for the PR, looks good, just one question for my own understanding.

File: clients/src/main/java/org/apache/kafka/clients/consumer/internals/AbstractCoordinator.java (hunk 1 above)
Comment (hachikuji) [reply clients/src/main/java/org/apache/kafka/clients/consumer/internals/AbstractCoordinator.java:655]: We check for interrupts in `ConsumerNetworkClient.poll()`, so it seemed unnecessary.

Comment (ijuma) [reply clients/src/main/java/org/apache/kafka/clients/consumer/internals/AbstractCoordinator.java:655]: Makes sense, thanks for the explanation.

**Review by ijuma — APPROVED** [review]
LGTM, merging to trunk.

**Pull request closed without merging**